pkg/cli/internal/commands/run/run.go

```diff
@@ -27,14 +29,19 @@ import (
 type runOptions struct {
 	general *options.GeneralOptions
 
-	debug    bool
-	filter   []string
-	notty    bool
-	pinMaps  string
-	pinProgs string
-	promPort uint32
+	debug        bool
+	filter       []string
+	histBuckets  []string
+	histValueKey []string
+	notty        bool
+	pinMaps      string
+	pinProgs     string
+	promPort     uint32
 }
 
+const histBucketsDescription string = "Buckets to use for histogram maps. Format is \"map_name,<buckets_limits>\"" +
+	"where <buckets_limits> is a comma separated list of bucket limits. For example: \"events:[1,2,3,4,5]\""
```

Comment:
Shouldn't this be?
```suggestion
	"where <buckets_limits> is a comma separated list of bucket limits. For example: \"events,[1,2,3,4,5]\""
```

Maybe we shouldn't require a `,` if we only have a single item in the list?

Reply:
You're right, good catch!

I think we'd still want the comma to separate the name of the map from the bucket limits wouldn't we? e.g. user would still pass:
```
--hist-buckets events,[1]
```